src/extensions/default/HealthData/HealthDataPreview.js

```diff
@@ -45,7 +45,11 @@ define(function (require, exports, module) {
         var result = new $.Deferred();
 
         HealthDataManager.getHealthData().done(function (healthDataObject) {
-            var content = JSON.stringify(healthDataObject, null, 4);
+            var combinedHealthAnalyticsData = HealthDataManager.getAnalyticsData("pingData", "", "", ""),
+                content;
+            combinedHealthAnalyticsData = [healthDataObject, combinedHealthAnalyticsData ];
+            combinedHealthAnalyticsData.join("\n\n");
```

Comment:
Sorry if I missed anything, but array join returns the joined elements as a string. Where it's received? I doubt `content` would ever be what it's intended to. It will be a serialized array representation and not the joined elements as a string!

Reply:
@swmitra  updated the code with the changes